examples/demo-signals/lwc/demoSignals/error-handler.js

```diff
@@ -0,0 +1,28 @@
+import { $signal, $computed, $effect } from "c/signals";
+
+const anySignal = $signal(0);
+
+$computed(
+  () => {
+    anySignal.value;
+    throw new Error("An error occurred during a computation");
+  },
+  {
+    errorHandler: (error) => {
+      console.error("error thrown from computed", error);
+      // Allows for a fallback value to be returned when an error occurs.
+      return 0;
```

Comment:
if a return value is not provided when an error happens what is the computed returning? does it keep the previous value or return null by default?